src/PageContent/Settings/OrganizationDetails.js

```diff
@@ -2,16 +2,68 @@ import React, { Component } from 'react';
 import Map from '../../Map/Map.js';
 import { AccountType } from '../../Enums';
 import { accountsRef, donatingAgenciesRef } from '../../FirebaseConfig.js';
+import moment from 'moment-timezone';
 
 class OrganizationDetails extends Component {
 
     constructor(props) {
         super(props);
         this.state = {
-            isEditing: false
+            isEditing: false,
+            times: [],
         };
+        this.convertTimes = this.convertTimes.bind(this);
+        this.dayRow = this.dayRow.bind(this);
+        this.dayNames = ['Sun','Mon','Tue','Wed','Thur','Fri','Sat'];
+
+
+    }
+
+    dayRow(i, day) {
+
+        var checkboxName = day + 'Check';
+        var startName = day + 'Start';
+        var endName = day + 'End';
+
+        // populate default values if exists
+        var checked = false;
+        var startTime = '';
+        var endTime = '';
+        var availabilities = this.props.org.availabilities;
+        if (availabilities && availabilities[i]) {
+            checked = true;
+            startTime = availabilities[i].startTimestamp;
+            endTime = availabilities[i].endTimestamp;
+            console.log('startTime:', startTime)
+        }
+
+        return (
+            <div className="row" key={day}> 
+                <input type="checkbox" name={checkboxName} defaultChecked={checked} />
+                <div className="day">{day}</div>
+                {/* TODO: AM/PM UI */}
+                <input type="time" name={startName} defaultValue={moment(startTime).format("HH:mm")} className="ra3-inputBox" />
+                <span className="ra3-spacing">to</span>
+                <input type="time" name={endName} defaultValue={moment(endTime).format("HH:mm")} className="ra3-inputBox" />
+            </div>
+        );
     }
 
+    convertTimes() {
+        var convert = []
+        var dayNames = ['Sun','Mon','Tue','Wed','Thur','Fri','Sat'];
+        for (var i = 0; i < this.props.org.availabilities.length; i++) {
+             if (this.props.org.availabilities[i]) {
+                convert.push(<li>{dayNames[i] + ' ' + moment(this.props.org.availabilities[i].startTimestamp).format("h:mm a") + '-' + moment(this.props.org.availabilities[i].endTimestamp).format("h:mm a")}</li>)
```

Comment:
One other comment I forgot to make, check out the console when opening the Settings page. Whenever you render lists in React they need to be passed a "key" prop so that React can keep track of them / re-render the correct ones when changed. If you don't have a key prop React will throw an error to the console. A fine key for this would probably just be something like i, so you would just do key={i} I believe